webpack.config.js

```diff
@@ -294,7 +294,10 @@ export default async (env, argv) => {
     const plainAliases = getIconPlainAliases(iconsDataBySlugs[icon.slug]);
 
     return {
-      guidelines: icon.guidelines,
+      guidelines:
+        typeof icon.guidelines != 'object'
+          ? icon.guidelines
+          : icon.guidelines?.trademark ? icon.guidelines.trademark : icon.guidelines.branding,
```

Comment:
As this part of the code will only be executed _if_ `guidelines` is an object then there shouldn't be a need to check that the `guidelines` exist.
```suggestion
          : icon.guidelines.trademark ? icon.guidelines.trademark : icon.guidelines.branding,
```